tests/test_web_urldispatcher.py

```diff
@@ -458,3 +459,34 @@ async def test_static_absolute_url(aiohttp_client: Any, tmp_path: Any) -> None:
     client = await aiohttp_client(app)
     resp = await client.get("/static/" + str(file_path.resolve()))
     assert resp.status == 403
+
+
+@pytest.mark.xfail(
+    raises=AssertionError,
+    reason="Regression in v3.7: https://github.com/aio-libs/aiohttp/issues/5621",
+)
+@pytest.mark.parametrize(
+    ("route_definition", "urlencoded_path", "expected_http_resp_status"),
+    (
+        ("/{user_ids:([0-9]+)(,([0-9]+))*}/hello", "/467%2C802%2C24834/hello", 200),
```

Comment:
Still need this case to be explicit
```suggestion
        ("/467,802,24834/hello", "/467%2C802%2C24834/hello", 200),
        ("/{user_ids:([0-9]+)(,([0-9]+))*}/hello", "/467%2C802%2C24834/hello", 200),
```

Reply:
I thought we decided we could keep regex for extra coverage

Reply:
ahh add another testcase, will do